components/automate-cli/pkg/verifyserver/constants/externalpostgresqlconstants.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+const (
+	EXTERNALPGSUCCESSCONNECTIONTITLE  = "Machine is able to connect with External Managed Postgres"
+	EXTERNALPGFAILCONNECTIONTITLE     = "External Postgresql Connection failed"
+	EXTERNALPGCONNECTIONERRORMSG      = "Machine is unable to connect with External Managed Postgresql"
+	EXTERNALPGCONNECTIONRESOLUTIONMSG = "Ensure that the Postgres configuration provided is correct. Review security group or firewall settings as well on the infrastructure"
```

Comment:
it should be 
EXTERNAL_PG_CONNECTION_RESOLUTION_MSG = "Ensure that the PostgreSQL configuration provided is correct. Review security group or firewall settings as well on the infrastructure"